nemo/collections/nlp/data/language_modeling/megatron/gpt_dataset.py

```diff
@@ -155,6 +164,10 @@ def get_indexed_dataset_(data_prefix, data_impl, skip_warmup):
 
 class GPTDataset(MegatronDataset):
     def __init__(self, cfg, trainer, name, data_prefix, documents, indexed_dataset, num_samples, seq_length, seed):
+        if not HAVE_APEX:
```

Comment:
Ok so this is done here, what about dataset utils, does it not have functions that use parallel_stare?

Reply:
it doesn't have `__init__`

Reply:
Ok. Hopefully anything using those functions will hard crash if apex is not found then.